More docs fixes. (#282)
osohq/oso: docs/more/faq.rst

We currently support Linux and Mac OS X.
We have initial Windows support, and expect publish a release for Windows soon.

Sign up for our newsletter in the footer anywhere on our docs if you'd like to
stay up to speed on the latest product updates.

--------------------------
What license does oso use?
--------------------------

.. |apache2| replace:: the *Apache 2.0 license*
.. _apache2: https://github.com/osohq/oso/blob/master/LICENSE

> oso licensed under |apache2|_.

Reviewer: ```suggestion
oso is licensed under |apache2|_.
```

----------------------
How does pricing work?
----------------------

> oso is freely available as an open source product and will always be open source. 

Reviewer: This makes it sound like it will always be open source but not necessarily always free. We should clarify that oso will always be open source **and** free.

> We are also working on an enterprise product, based on oso, which will come with
> additional features. If you are interested in support for oso or enterprise

Reviewer: ```suggestion
We are also working on an enterprise product that extends the open source core with
additional features. If you are interested in support for oso or enterprise
```

features, please `contact us <https://osohq.com/company/contact-us>`_.

----------------------------
Who builds and maintains oso
----------------------------

> oso is built by the eponymous company, oso. We are a Sequoia-backed company
> headquartered in New York City, with engineers across 3 time zones, and we are

Reviewer: ```suggestion
oso is built by oso! We are headquartered in New York City with engineers across 3 time zones, and we are
```
would also be cool to link the second "oso" to https://www.osohq.com/company/about-us

> hard at work on new features and improvements. If you have feedback or ideas on
> ways we can make the product better, we would be delighted to hear from you.

Reviewer: ```suggestion
hard at work on new features and improvements. If you have feedback or ideas about
how we can make the product better, we would be delighted to hear from you.
```
